src/components/Color/Color.js

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { Component } from 'react';
+import namer from 'color-namer';
 
-export function Color({hex}) {
+export class Color extends Component {
 
-  let style = { backgroundColor: hex };
+  determineLightness(hex) {
+    const color = hex.substring(1);
+    const rgbColor = parseInt(color, 16);
 
-  return (
-    <div style={style} className="Color">
-      { hex }
-    </div>
-  )
+    const red = (rgbColor >> 16) & 0xff;
+    const green = (rgbColor >> 8) & 0xff;
+    const blue = (rgbColor >> 0) & 0xff;
+
+    const lightLevel = 0.2126 * red + 0.7152 * green + 0.0722 * blue; 
```

Comment:
This is a particularly confusing formula that determines how light/dark a color is